### DISCUSSION TITLE
Add custom icons

### DISCUSSION THREAD

**Discussion by sl-pk** [zixpo/candybar#179] [General]
I've been having issues while trying to add my own icons. I am using the sample version.

- I cannot edit the `drawable.xml` file. If I edit anything in it it will undo once I run the application.

**Comment by Donnnno** [comment]
Are you sure that you're using the sample instead of the library? 
https://github.com/zixpo/candybar-sample

**Reply by sl-pk** [reply]
Yes, I've downloaded the sample project from the Wiki.

**Reply by kshitijdeota** [reply]
Use the other drawable.xml you may find in the "res" folder. Do not edit the files in "assets", xml files here is copied from "res" on every compile.

**Reply by sl-pk** [reply]
Thanks, that fixed the problem!
